src/hist/plot.py

```diff
@@ -261,20 +265,27 @@ def plot_pull(
         )
     yerr = np.sqrt(variances)
 
-    if type(func) in [str]:
+    if isinstance(func, str):
```

Comment:
Never check the type exactly, always use `isinstance`. It supports subclassing and MyPy type narrowing.

Reply:
Also never make a list for containership testing, use a set, it's faster. `x in {a, b}`.